src/overlays/actors/ovl_En_Encount4/z_en_encount4.h

```diff
@@ -13,7 +13,7 @@ typedef struct EnEncount4 {
     /* 0x0148 */ char unk_148[0x4];
     /* 0x014C */ s16 unk_14C;
     /* 0x014E */ s16 unk_14E;
-    /* 0x0150 */ char unk_150[0x8];
+    /* 0x0150 */ s16 unk_150;
 } EnEncount4; // size = 0x158
```

Comment:
```suggestion
    /* 0x0150 */ s16 unk_150;
    /* 0x0152 */ char unk_152[0x6];
} EnEncount4; // size = 0x158
```
Don't forget to add the padding to make the size the same.